Loop/AppDelegate.swift

```diff
@@ -91,4 +91,34 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, WindowProvider {
     func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
         return loopAppManager.supportedInterfaceOrientations
     }
+    
+    // MARK: - UIApplicationDelegate - handle external URLs
+
+    func application(_ application: UIApplication,
+                     open url: URL,
+                     options: [UIApplication.OpenURLOptionsKey : Any] = [:] ) -> Bool {
+
+        // Process the URL.
+        guard let components = NSURLComponents(url: url, resolvingAgainstBaseURL: true), let linkTarget = components.host else {
```

Comment:
I'd prefer to not include any of this example/aspirational code yet. It'd be better to be able to evaluate it in the context of an actual feature that was using it.